src/lib/y2partitioner/clients/main.rb

```diff
@@ -1,17 +1,34 @@
+require "y2partitioner/widgets/overview"
+require "y2storage"
+
 Yast.import "CWM"
 Yast.import "Stage"
 Yast.import "Wizard"
 
 module Y2Partitioner
   module Clients
+    # Main entry point to see partitioner configuration
     class Main
       extend Yast::I18n
       extend Yast::UIShortcuts
 
       def self.run
         textdomain "storage"
 
-        contents = HBox(
+        staging = Y2Storage::StorageManager.instance.y2storage_staging
```

Comment:
Why does this client work on staging instead of receiving the initial devicegraph as an argument?